miso-web/src/main/webapp/scripts/freezer_ajax.js

```diff
@@ -111,7 +111,8 @@
   Freezer.addShelfStorage = function(shelf) {
     Utils.showWizardDialog('Add Shelf Storage', [
         makeHandler('Rack', [depthField, heightField, barcodeField], '/shelves/' + shelf.id + '/racks'),
-        makeHandler('Stack', [heightField, barcodeField], '/shelves/' + shelf.id + '/stacks'), {
+        makeHandler('Stack', [heightField, barcodeField], '/shelves/' + shelf.id + '/stacks'),
+        makeHandler('Tray Rack', [heightField, barcodeField], '/shelves/' + shelf.id + '/tray-racks'), {
```

Comment:
The height field displays "Height (boxes):". In this case it might make sense to display "Height (trays):"